tests/test_cgnsutilities.py

```diff
@@ -115,3 +115,55 @@ def test_getNumNodes(self, train=False):
 
     def train_getNumNodes(self):
         self.test_getNumNodes(train=True)
+
+
+class TestReturnFuncs(unittest.TestCase):
+    def setUp(self):
+        # Use the same grid file with different names
+        self.grid1 = readGrid(os.path.abspath(os.path.join(baseDir, "../examples/717_wl_L2.cgns")))
+        self.grid1.name = "grid1"
+        self.grid2 = readGrid(os.path.abspath(os.path.join(baseDir, "../examples/717_wl_L2.cgns")))
+        self.grid2.name = "grid2"
+        self.grids = [self.grid1, self.grid2]
+
+    def test_combineGrids(self, train=False):
+
+        combinedNewNames = combineGrids(self.grids)
+        combinedOldNames = combineGrids(self.grids, useOldNames=True)
+
+        newNames = [blk.name for blk in combinedNewNames.blocks]
+        oldNames = [blk.name for blk in combinedOldNames.blocks]
+
+        # Blocks are named after the grid name
+        self.assertEqual(
+            newNames,
+            [
+                "grid1.00001",
+                "grid1.00002",
+                "grid1.00003",
+                "grid1.00004",
+                "grid1.00005",
+                "grid2.00001",
+                "grid2.00002",
+                "grid2.00003",
+                "grid2.00004",
+                "grid2.00005",
+            ],
+        )
+
+        # Block names from original grids are preserved
```

Comment:
Is it typical to do both of these in the same test as opposed to separate tests? I would think for unit testing we would want these in separate tests